pages/docs/2_modify/advanced/extending_local_version_comunica.md

```diff
@@ -0,0 +1,52 @@
+---
+title: 'Using Local Changes to Base Comunica in Comunica Extensions'
+description: 'Guide on how to extend a local version of Comunica or use a local version of Comunica with an existing extension'
+---
+
+Various  [extensions](https://comunica.dev/docs/modify/extensions/) of Comunica exist. 
```

Comment:
This is a good motivation for linking indeed, but I would suggest adding a sentence before saying that it may be required to link comunica to another project (in general). Linking to a comunica extension is one good example where this is valuable, but it may also be necessary to link a dev-version of comunica into other projects, in which case this same method can be used.

Reply:
Also emphasize in the description that this approach does not require publishing anything to npm.
So it's useful for testing things _before_ they are published.